botan-src/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+use std::env;
+use std::path::PathBuf;
+use std::process::Command;
+
+const BUILD_ERROR_MSG: &str = "Unable to build botan.";
+const SRC_DIR_ERROR_MSG: &str = "Unable to find the source directory.";
+const SRC_DIR: &str = "botan";
+const INCLUDE_DIR: &str = "build/include/botan";
+
+macro_rules! pathbuf_to_string {
+    ($s: ident) => {
+        $s.to_str().expect(BUILD_ERROR_MSG).to_string()
+    };
+}
+
+fn configure(build_dir: &str) {
+    let mut configure = Command::new("python");
+    configure.arg("configure.py");
```

Comment:
This seems to be ignoring cross compilation - the build will be configured for whatever the underlying host system is.

Reply:
I just added support for configuration options via environment variables. Cross compilation can now be used this way.